### ISSUE TITLE
Service system_log/write called

### ISSUE THREAD

**Issue by gieljnssns** [matthinc/HomeAssistantElectron#17]
Since 0.65.0 I get this
```
2018-03-12 08:17:08 ERROR (MainThread) [frontend.js.es5.201803100] /Applications/Home Assistant.app/Contents/Resources/app/src/preload.js:22:11 Uncaught TypeError: Cannot read property 'querySelector' of null

```
5 times every second
when I close the electron app this stops

**Comment by rudders** [comment]
+1 from me - I’ll try and figure out some more debug info and update this issue.

**Comment by matthinc** [comment]
Thank you for the report! I already figured out the problem and fixed it: https://github.com/matthinc/HomeAssistantElectron/commit/a0e4af6ac9231134fd454def08cbdc29dbdb2c55

HomeAssistant changed a few things in its DOM for 0.65. The problem is, that the fix would break all versions before 0.65. Therefore I want to wait a bit, until 0.65 is installed by most of the users. As a quick workaround, you could just uncheck the option 'Don't hide toolbar when scrolling' in the settings.

On a long-term perspective I'll have to develop some sort of version detection.

**Comment by rudders** [comment]
Cool - nice one!
